Framework/API/test/FunctionFactoryTest.h

```diff
@@ -433,9 +433,9 @@ class FunctionFactoryTest : public CxxTest::TestSuite {
   void test_MultiDomainFunction_creation_moreComplex() {
     const std::string fnString =
         "composite=MultiDomainFunction,NumDeriv=true;(name=FunctionFactoryTest_"
-        "FunctA,a0=0,a1=0.5;name=FunctionFactoryTest_FunctB,b0=0.1,b1=0.2,ties="
+        "FunctA,a0=0,a1=0.5;name=FunctionFactoryTest_FunctB,b0=0.1,ties="
```

Comment:
Okay I see. I just to check that changing the string format isn't going to affect old scripts or saved nexus files or anything?